docs/reference/createPlugin-feature-flags.md

```diff
@@ -0,0 +1,65 @@
+# createPlugin - feature flags
+
+The `featureFlags` object passed to the `register` function makes it possible for plugins to register Feature Flags in Backstage for users to opt into. You can use this to split out logic in your code for manual A/B testing, etc.
+
+```typescript
+export type FeatureFlagsHooks = {
+  register(name: FeatureFlagName): void;
```

Comment:
Technically this should be `registerInSidebar` or something more verbose. It technically doesn't (and I don't think it will) serve any other purpose than for displaying a list of all available feature flags.